prometheus/registry.go

```diff
@@ -505,22 +470,30 @@ func (r *registry) writePB(encoder expfmt.Encoder) error {
 		sort.Sort(metricSorter(mf.Metric))
 	}
 
-	// Write out MetricFamilies sorted by their name.
+	// Write out MetricFamilies sorted by their name, skipping those without
+	// metrics.
```

Comment:
These are exposed in other clients. I'd see it as useful for spotting where you've not exposed any labels for a vector yet.